services/openuserjs/openuserjs-downloads.tester.js

```diff
@@ -7,5 +7,5 @@ t.create('Downloads')
   .expectBadge({ label: 'downloads', message: isMetric })
 
 t.create('Downloads (not found)')
-  .get('/dt/NotAUser/NotAScript.json')
+  .get('/dt/NotAUser/NotAScript1.json')
```

Comment:
Okay... presuming that you are polling the site already in these lines *(based off the commit summary)* you might be asking for a headache with picking `NotAUser`. Someone could actually make this account and you might be introducing a random element in shields testing routines. Usually I do my test routines against my own account but that is up to Shields on whether or not they want to allow that. As long as you let me know you could use my username as I am relatively sure I'll never publish a "NotAScript" phantom script... or your can choose your own account... either way someone, in its current state, could easily make the tests fail and someone will be barking on Shields about it.

Reply:
Sounds good to me, I'd be fine with setting it to use my own account

Reply:
> I suppose for a similar reason, the other tests should probably be changed to scripts that I know won't be changing randomly

Yeah I would do that as well since those too use `.get` *(which I still presume is actually polling OUJS)* NatoBoram is active but doesn't seem to be available at this time. If you are absolutely set in your way of showcasing his script... fork it on OUJS by going to his source code page and Submit Code as Fork and it will back-reference the original. You can steer others towards the root script in your Script Info.

I always do the Meta View Unit Test script in examples as I know unless something catastrophic occurs with Userscripts it should always be there... that is another choice.